### PR TITLE
Bump to 0.12.6

### PR DESCRIPTION
**Pull request by hadim**
Also add myself as a maintainer.

### CODE REVIEW

**Comment by hadim** [comment]
@oblute Any chance you could review please?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,24 +23,23 @@ requirements:
     - pip
     - python >=3.6
   run:
-    - click >=7.0
+    - click >=7.0,!=8.0.0
```
Comment (mariusvniekerk) [inline recipe/meta.yaml:26]: sure this shouldn't be <8? presumably when 8.0.1 that still won't work for this version

Comment (hadim) [reply recipe/meta.yaml:26]: It's what the upstream package indicates but I think you're right it's safest to set <8.

**Comment by hadim** [comment]
LGTM

**Comment by vanpelt** [comment]
@mariusvniekerk I'm one of the founders of wandb.  We would love to find a way to update this feedstock whenever we cut new releases to pypi.  Would that be something you would be open to?

**Comment by mariusvniekerk** [comment]
@vanpelt you can add yourself as a maintainer to this feedstock

**Comment by andrewtruong** [comment]
Hi, just curious why your `meta.yaml` looks so different from the default one generated by `conda skeleton`?  Namely `requirements.host` and `test`.

Here is what I got when I ran `conda skeleton`
```yaml
{% set name = "wandb" %}
{% set version = "0.12.7" %}

package:
  name: "{{ name|lower }}"
  version: "{{ version }}"

source:
  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
  sha256: d1c2a6cd3fc57b5a75ff18d4bb69f088d0bd1d0ad22c98005a37e037c85442c7

build:
  number: 0
  entry_points:
    - wandb=wandb.cli.cli:cli
    - wb=wandb.cli.cli:cli
  script: "{{ PYTHON }} -m pip install . -vv"

requirements:
  host:
    - click >=7.0,!=8.0.0
    - configparser >=3.8.1
    - docker-pycreds >=0.4.0
    - gitpython >=1.0.0
    - pathtools
    - pip
    - promise >=2.0,<3
    - protobuf >=3.12.0
    - psutil >=5.0.0
    - python
    - python-dateutil >=2.6.1
    - pyyaml
    - requests >=2.0.0,<3
    - sentry-sdk >=1.0.0
    - shortuuid >=0.5.0
    - six >=1.13.0
    - subprocess32 >=3.5.3
    - yaspin >=1.0.0
  run:
    - click >=7.0,!=8.0.0
    - configparser >=3.8.1
    - docker-pycreds >=0.4.0
    - gitpython >=1.0.0
    - pathtools
    - promise >=2.0,<3
    - protobuf >=3.12.0
    - psutil >=5.0.0
    - python
    - python-dateutil >=2.6.1
    - pyyaml
    - requests >=2.0.0,<3
    - sentry-sdk >=1.0.0
    - shortuuid >=0.5.0
    - six >=1.13.0
    - subprocess32 >=3.5.3
    - yaspin >=1.0.0

test:
  imports:
    - wandb
  commands:
    - wandb --help
    - wb --help
  requires:
    - mock >=2.0.0
    - tox-pyenv >=1.0.3

about:
  home: "https://github.com/wandb/client"
  license: MIT
  license_family: MIT
  license_file: 
  summary: "A CLI and library for interacting with the Weights and Biases API."
  doc_url: 
  dev_url: 

extra:
  recipe-maintainers:
    - your-github-id-here

```

**Pull request merged**